tests/TestCTimers.cpp

```diff
@@ -0,0 +1,129 @@
+#include "TestCTimers.h"
```

Comment:
License lines should be added above #include too

// SPDX-License-Identifier: GPL-2.0-or-later
// Copyright (C) 2023 The MMapper Authors